modules/verify-acs-installation-ocp.adoc

```diff
@@ -45,9 +45,9 @@ $ oc new-project test
 [source,terminal]
 ----
 $ oc run shell --labels=app=shellshock,team=test-team \
-  --image=vulnerables/cve-2014-6271 -n test
+  --image=vulnerables/example-vulnerables-cve-2014-6271 -n test
```

Comment:
You need to use the full image path here: `quay.io/stackrox-io/docs:example-vulnerables-cve-2017-7494`
`quay.io/stackrox-io/docs:example-vulnerables-cve-2014-6271`